storage/client/file/src/main/java/com/azure/storage/file/models/FileRange.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.models;
+
+/**
+ * The range of a file in the storage file service.
+ */
+public final class FileRange {
+    private long start;
+    private long end;
+
+    /**
+     * Create an instance of the range of a file.  Both the start and end of the range must be specified.
+     * @param start Specifies the start of bytes to be written.
+     * @param end Specifies the end of bytes to be written
+     */
+    public FileRange(final long start, final long end) {
```

Comment:
Based on the [documentation](https://docs.microsoft.com/en-us/rest/api/storageservices/specifying-the-range-header-for-file-service-operations) `end` is optional, we should turn it into a `Long`

Reply:
Good catch

Reply:
Done.